src/metabase/email/pulse.mustache

```diff
@@ -1,8 +1,17 @@
 {{> metabase/email/_header}}
   {{#pulseName}}
-    <h1 style="{{sectionStyle}} margin: 16px; color: {{color-brand}};">
-      {{pulseName}}
-    </h1>
+    {{#pulseLink}}
+      <a href="{{pulseLink}}">
+        <h1 style="{{sectionStyle}} margin: 16px; color: {{color-brand}};">
+          {{pulseName}}
+        </h1>
+      </a>
+    {{/pulseLink}}
+    {{^pulseLink}}
+      <h1 style="{{sectionStyle}} margin: 16px; color: {{color-brand}};">
+        {{pulseName}}
+      </h1>
+    {{/pulseLink}}
```

Comment:
I don't like how not-DRY this is, but AFAIK there's not a better way in Mustache

Reply:
yeah I don't think you can really do any better than this unfortunately